### PR TITLE
Add compoments file

### PR DESCRIPTION
**Pull request by Ngufor-emmanuella**
In this project I built a calculator app using REACT by implementing the following :
-Use React components.
-Use React props.
-Use React state.
-Delete all the boilerplate from CRA (text, images, styles).
-create a directory called components.
-Inside components, create a new Calculator.js file.
-In Calculator.js, create a React component that matches the required design

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                <button type="button" className="no-color" onClick={this.handPress}> AC </button>
+              </td>
+              <td>
+                <button type="button" className="no-color" onClick={this.handPress}> +/- </button>
+              </td>
+              <td>
+                <button type="button" className="no-color" onClick={this.handPress}> % </button>
+              </td>
+              <td>
+                <button type="button" className="color" onClick={this.handPress}> + </button>
```
Comment (TedLivist) [inline src/components/calculator.js]: - [ ] From the design provided this button should be the divide sign, not plus. Kindly fix that.

  If you do not have the divide sign on your PC you can use the HTML entity code for this: **`&#xf7;`**

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Ngufor-emmanuella,

Good job so far!
✔️ Class-based components
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.js (hunk 1 above)
Comment (Ngufor-emmanuella) [reply src/components/calculator.js]: thank you

**Review by julie-ify — APPROVED** [review]
Hi @Ngufor-emmanuella,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ngufor-emmanuella** [comment]
thank you

**Pull request merged**